### ISSUE TITLE
`pymc.sample` deprecated after v5.10.0

### ISSUE THREAD

**Issue by HumphreyYang** [QuantEcon/lecture-python.myst#380]
Hi @mmcky,

`pymc.sample` is deprecated after v5.10.0 as shown [here](https://www.pymc.io/projects/docs/en/v5.10.0/api/generated/pymc.sample.html#pymc-sample).

**Comment by HumphreyYang** [comment]
Hi @mmcky,

Strangely, after testing it under `pymc-5.10.2`, I found it is removed from the documentation but it can still be called. 

[`pymc.sampling.mcmc.sample`](https://www.pymc.io/projects/docs/en/latest/api/generated/pymc.sampling.mcmc.sample.html#pymc-sampling-mcmc-sample) does the same thing, perhaps it is calling `pymc.sampling.mcmc.sample` at the backend.

Would you still like me to update it?

**Comment by mmcky** [comment]
@HumphreyYang may we should report this to the `pymc` project?

**Comment by mmcky** [comment]
@HumphreyYang would you mind to update this

The link in `ar1_bayes` is not working as indicated above.

https://www.pymc.io/projects/docs/en/stable/projects/docs/en/v5.10.0/api/generated/pymc.sampl.html#pymc-sample

**Comment by HumphreyYang** [comment]
Thanks @mmcky, I will do it. 

I submitted a PR to `pymc` a year ago but it still hasn't been merged : )

**Comment by mmcky** [comment]
Roger that. 

Has the routine been renamed due to deprecation? Should we be using a different method?

**Comment by HumphreyYang** [comment]
> Roger that.
> 
> Has the routine been renamed due to deprecation? Should we be using a different method?

No, they updated the structure of their documentation (unintentionally), the methods are not changed : )

**Comment by mmcky** [comment]
@HumphreyYang is this still relevant?

I see the original link is still available in `latest`

https://www.pymc.io/projects/docs/en/latest/api/generated/pymc.sample.html

**Comment by HumphreyYang** [comment]
They haven't merged my PR yet so I assume that they are not going to fix it. 

Perhaps let's close this for now.
